src/cdk/coercion/coercion.md

```diff
@@ -1,3 +1,59 @@
-### Coercion
-
 Utility functions for coercing `@Input`s into specific types.
+
+### Example
+
+```ts
+import {Directive, ElementRef, HostListener} from '@angular/core';
+import {
+  coerceBooleanProperty,
+  BooleanInput,
+  NumberInput,
+  coerceNumberProperty,
+  coerceElement,
+} from '@angular/cdk/coercion';
+
+@Directive({
+  selector: 'my-button',
+  host: {
+    '[disabled]': 'disabled',
+    '(click)': 'greet()',
+  }
+})
+class MyButton {
+  // Using `coerceBooleanProperty` allows for the disabled value of a button to be set as
+  // `<my-button disabled></my-button>` instead of `<my-button [disabled]="true"></my-button>`.
+  // It also allows for a string to be passed like `<my-button disabled="true"></my-button>`.
+  @Input()
+  get disabled() { return this._disabled; }
+  set disabled(value: any) {
```

Comment:
Sorry to ask, but is `any` correct here (and in the other example)? I ask therefore, because as it is a bad practice, IMHO it shouldn't be in official docs as this can influence bad habits for other devs. Note that a simple boolean would solve it (as you already does in majority of your components).